aten/src/ATen/core/function_schema.h

```diff
@@ -33,6 +35,9 @@ struct Argument {
   const std::string& name() const {
     return name_;
   }
+  const c10::optional<std::string>& field_name() const {
```

Comment:
Adding an additional `field_name` here is messy because it is not used in arguments. Can we just use the `name()` field? It is unused for return values anyway. To distinguish between a present an non-present field name, we can just use "" for unnamed returns.